rest/logger/cloudwatch.go

```
@@ -0,0 +1,65 @@
+package logger
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/RedHatInsights/chrome-service-backend/config"
+	"github.com/aws/aws-sdk-go/aws"
```

golangci-lint (2.12.2)

[error] SA1019: "github.com/aws/aws-sdk-go/aws" is deprecated: aws-sdk-go is deprecated. Use aws-sdk-go-v2. See https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-aws-sdk-for-go-v1-on-july-31-2025/. 

(staticcheck)